### ISSUE TITLE
Habroxia 2 - CUSA25058

### ISSUE THREAD

**Issue by and468860** [red-prig/fpps4-game-compatibility#1189]
![Gravar_2023_01_19_12_49_09_229 mp4_snapshot_00 03 537](https://user-images.githubusercontent.com/61087055/213449738-377157b1-21d0-49e7-b5c6-c771a901046e.jpg)

**Comment by and468860** [comment]
Not status inhame  gameplay

**Comment by ArbestRi** [comment]
Yes, but it slows down over time. Maybe if @red-prig makes optimizations it will be better.

**Comment by gandalfthewhite19890404** [comment]
I guess - 4 fps is not playable, I can add additional status-playable, but it is not true.

**Comment by and468860** [comment]
to play in the game by reading

**Comment by yaya54840** [comment]
``
![Capture d'écran_20231229_170410](https://github.com/red-prig/fpps4-game-compatibility/assets/47884186/9fc5d134-5edc-485c-96aa-2ad8963c1753)
hello Gandalph This game is running at full speed with image load hack
With 905a9a6 and version 1.12
So Playable
